pkg/cli/images.go

```diff
@@ -61,10 +62,10 @@ func (a *Image) Run(cmd *cobra.Command, args []string) error {
 		return printContainerImages(images, cmd.Context(), c, a)
 	}
 
-	out := table.NewWriter(tables.Image, system.UserNamespace(), false, a.Output)
+	out := table.NewWriter(tables.ImageAcorn, system.UserNamespace(), false, a.Output)
 	if a.Quiet {
 		out = table.NewWriter([][]string{
-			{"Name", "{{ . | name }}"},
+			{"Name", "{{ .Name }}"},
```

Comment:
Do you know why this had to change?

Reply:
was getting this error with the intermediate imagePrint object 
```
Error: template: :1:7: executing "" at <name>: error calling name: invalid obj cli.imagePrint
```
intermediate image is for separating out each tag as an individual print line

Reply:
ok